apps/docs/src/app/platform/component-docs/platform-input/platform-input-docs.component.html

```diff
@@ -0,0 +1,17 @@
+<fd-docs-section-title [id]="'inputdef'" [componentName]="'input'">
+	Input
+</fd-docs-section-title>
+<description>This example shows Input default implementaion </description>
```

Comment:
This doesn't appear on the Preview page. 

Can you also please provide more description for each category of examples, like in the Popover component you reference below?